packages/filesystem/src/common/filesystem.ts

```diff
@@ -105,6 +105,11 @@ export interface FileSystem extends JsonRpcServer<FileSystemClient> {
      */
     getEncoding(uri: string): Promise<string>;
 
+    /**
+     * Guess encoding of a given file besed on its content.
+     */
+    guessEncoding(uri: string): Promise<string | undefined>;
```

Comment:
we should update `updateContent` method here that it accepts a new options `overwriteEncoding` and document what is the difference to `encoding` and how it should be used.

Reply:
> we should update `updateContent` method here that it accepts a new options `overwriteEncoding` and document what is the difference to `encoding` and how it should be used.

@akosyakov `updateContent` method and document updated.